### PR TITLE
Switch to 3.3 stable

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -44,23 +44,23 @@
...
+        "symfony/dependency-injection": "^3.3",
         "symfony/doctrine-bridge": "^2.8 || ^3.0",
-        "symfony/event-dispatcher": "^3.3@beta",
+        "symfony/event-dispatcher": "^3.3",
         "symfony/expression-language": "^2.8 || ^3.0",
-        "symfony/finder": "^3.3@beta",
-        "symfony/form": "^3.3@beta",
-        "symfony/framework-bundle": "^3.3@beta",
+        "symfony/finder": "^3.3",
+        "symfony/form": "^3.3",
```
Comment (meyerbaptiste) [inline composer.json]: Should be removed.

**Comment by meyerbaptiste** [comment]
I bumped a lot of dependencies because of stability requirements but should we not downgrade to older versions?
* `"symfony/http-foundation": "^3.1"`
* `"symfony/asset": "^2.7 || ^3.0"`
* `"symfony/cache": "^3.1"`
* `"symfony/config": "^3.2"`
* `"symfony/event-dispatcher": "^2.7 || ^3.0"`
* `"symfony/finder": "^2.7 || ^3.0"`
* etc.

Then, IMO, a dev dependency is missing:
* `"symfony/debug": "^2.8 || ^3.0"`, because we use the `Symfony\Component\Debug\Exception\FlattenException` exception.

**Comment by Simperfit** [comment]
I agree with @meyerbaptiste we could keep some low deps

**Comment by dunglas** [comment]
Done @meyerbaptiste @Simperfit

**Pull request merged**